policy/release/commit_signature_test.rego

```diff
@@ -0,0 +1,33 @@
+package policy.release.commit_signature
+
+import data.lib
+
+mock_data(signatures) = d {
+    d := {"body": {"Hash": ["nm32n5235"]},"signatures": signatures}
```

Comment:
(nit) the structure could be simpler, something like:

```json
{
  "<commit-id1>": ["email1", "email2", ...],
  "<commit-id2>": ["email3", "email4", ...],
}
```

Reply:
Just for my own understanding, why is `Hash` an array instead of a single string?